model/riscv_pmp_regs.sail

```diff
@@ -100,6 +100,15 @@ bitfield Pmpcfg_ent : bits(8) = {
   R : 0         /* read */
 }
 
+bitfield Mseccfg_ent : xlenbits = {
+  RLB : 2,        /* Rule Locking Bypass */
+  MMWP : 1,       /* Machine Mode Whitelist Policy */
+  MML : 0         /* Machine Mode Lockdown */
+}
+
+register mseccfg  : Mseccfg_ent
+register mseccfgh  : bits(32)
```

Comment:
What's with the spaces? Line up or single space, don't do multiple spaces that don't line up.